packages/core/src/app/utils/merkle-tree.ts

```diff
@@ -161,48 +126,28 @@ export class OptimizedSparseMerkleTree implements SparseMerkleTree {
     })
   }
 
-  public async update(key: BigNumber, value: Buffer): Promise<boolean> {
+  public async update(leafKey: BigNumber, leafValue: Buffer): Promise<boolean> {
```

Comment:
I already mentioned this in our call but of course long term we'll probably want 1) to use native `bigint` in the backend of our BigNumber library / replace with bitint directly / replace with buffer directly.

That said, none of these are worth the effort pre-devocon.